lua/lspconfig/configs/clojure_lsp.lua

```diff
@@ -4,7 +4,13 @@ return {
   default_config = {
     cmd = { 'clojure-lsp' },
     filetypes = { 'clojure', 'edn' },
-    root_dir = util.root_pattern('project.clj', 'deps.edn', 'build.boot', 'shadow-cljs.edn', '.git', 'bb.edn'),
+    root_dir = function(pattern)
+      local fallback = vim.loop.cwd()
```

Comment:
does single_file_support not work with this server?

Reply:
It actually works, I'll switch to this option.